contracts/contracts/strategies/balancer/BaseBalancerStrategy.sol

```diff
@@ -481,13 +510,62 @@ abstract contract BaseBalancerStrategy is InitializableAbstractStrategy {
 
     function _approveBase() internal virtual {
         IERC20 pToken = IERC20(platformAddress);
+
         // Balancer vault for BPT token (required for removing liquidity)
-        pToken.safeApprove(address(balancerVault), type(uint256).max);
+        // slither-disable-next-line unused-return
+        pToken.approve(address(balancerVault), 0);
```

Comment:
nit: Any reason for removing the usage of `safeApprove` but still doing the same thing it was gonna do (i.e. set to 0 before setting it to another value)??

Reply:
I've removed it because it is deprecated: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v4.4/contracts/token/ERC20/utils/SafeERC20.sol#L39-L58

Here is some more data on why that is so: https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2219

Also safeApprove didn't actually set the balance to 0 and then to the requested one, rather just checked that the allowance was non zero before setting it to some non zero value.